dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExclusionFileParser.java

```diff
@@ -62,12 +68,29 @@
   static ImmutableList<LinkageErrorMatcher> parse(Path exclusionFile)
       throws SAXException, IOException {
 
+    File exclusion = exclusionFile.toFile();
+    validate(exclusion);
+
     XMLReader xmlReader = XMLReaderFactory.createXMLReader();
     ExclusionFileHandler handler = new ExclusionFileHandler();
     xmlReader.setContentHandler(handler);
 
-    InputSource inputSource = new InputSource(new FileInputStream(exclusionFile.toFile()));
+    InputSource inputSource = new InputSource(new FileInputStream(exclusion));
     xmlReader.parse(inputSource);
```

Comment:
If you call `inputSource.setSystemId(path.toURI())` or some such thing here, error reporting will improve.

Reply:
Updated.